docs/jwst/straylight/reference.rst

```diff
@@ -1,20 +1,25 @@
 Reference File
 ==============
-The MIRI MRS straylight correction step uses a straylight mask. There
+The default MIRI MRS stray-light correction step uses Regions reference file. This
+Regions files is the first extension in the MIRI Distortion file and has  been read
```

Comment:
Oh wait, so it's not a separate reference file, but is actually just information in the first extension of the Distortion reference file? In that case, you should not say that this step uses the Regions reference file, but instead it uses the Distortion reference file.